python/cuml/test/test_random_forest.py

```diff
@@ -834,3 +834,17 @@ def test_concat_memory_leak(large_clf, estimator_type):
     logger.info("Final memory delta: %d" % (
         (used_mem - initial_baseline_mem)/1e6))
     assert (used_mem - initial_baseline_mem) < 1e6
+
+
+@pytest.mark.xfail(strict=True, raises=ValueError)
```

Comment:
I think it's generally nicer to use:
```
with pytest.raises(ValueError):
  fit
```
since that is more specific about the code raising and clearer to the reader. Can wait for future refactor PR  though.